.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.0.0
```

Comment:
I can also add this to a different PR if preferred. I think it's important to keep track of node versions somewhere, especially for those who are utilizing version tracking systems where their plugins (like CoC) rely on them.